app/goal_routes.py

```diff
@@ -0,0 +1,114 @@
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+import requests
+from app.helper_routes import get_one_obj_or_abort
+
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+#--------------------------------POST-------------------------------
+@goal_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    
+    if "title" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Nice that you're doing input validation here, but it would be good to let the user know what data was invalid.